File: coding-ig/tahun-kabisat/tahun-kabisat.R
n <- as.integer(readline(promp="Masukkan Tahun = "))                                    
if (n%%400 == 0){                                                            
   cat(sprintf("%d adalah tahun Kabisat",n))                                    
}else if (n%%100 == 0){                                                      
   cat(sprintf("%d bukan tahun Kabisat",n))                                     
}else if (n%%4 == 0){                                                        
   cat(sprintf("%d adalah tahun Kabisat",n))                                    
}else{                                                                      
   cat(sprintf("%d bukan tahun Kabisat",n))                                     
}   

File: coding-ig/tahun-kabisat/tahunkabisat.m
n = input("Masukkan Tahun = ");
if (mod(n,400) == 0)
  printf("%d adalah tahun Kabisat",n);
elseif (mod(n,100) == 0)
    printf("%d bukan tahun Kabisat",n);
elseif (mod(n,4) == 0)
    printf("%d adalah tahun Kabisat",n);
else
    printf("%d bukan tahun Kabisat",n);
end